test/conf/cassandra-mtls.yaml

```diff
@@ -87,3 +86,5 @@ authenticator:
   class_name : org.apache.cassandra.auth.MutualTlsAuthenticator
   parameters :
     validator_class_name: org.apache.cassandra.auth.SpiffeCertificateValidator
+accord:
+  journal_directory: build/test/cassandra/accord_journal
```

Comment:
FYI, you can also do

```
accord.journal_directory: build/test/cassandra/accord_journal
```

No need to change, was mostly saying as it might have been faster for you when you first changed these, so thought to mention

Reply:
yeah, but now we have all the Accord related stuff grouped together